[Define] Modify enum type to enum class

jieon-seol opened the pull request:
변경내용:
  enum 을 enum class로 변경
  OPTION으로 하나로 되어있던 것을 OPTION1,2,3으로 분리함
  OPTION2에 세부 옵션별 enum 추가함
  OPTION1,2,3, OPERATION_TYPE에 NONE(-1)과 MAX를 추가함

sky64422 commented on dynamic/Define.h:
```diff
@@ -1,8 +1,10 @@
-enum OPERATION_TYPE { ADD, DEL, SCH, MOD };
+enum class OPTION1 { NONE = -1, P, MAX };
+enum class OPTION2 { NONE = -1, Y, M, D, L, F, MAX };
+enum class OPTION3 { NONE = -1, MAX_OPTION3 };
```
여기도 위 아래의 enum 과 형식을 통일시킨다면, MAX_OPTION3 -> MAX 를 쓰는 게 어떤가요??

jieon-seol replied:
넵 수정하겠습니다

jieon-seol committed:
[Define] Modify enum type to enum class

- devide OPTION to OPTION1, OPTION2, OPTION3
- add NONE, MAX to OPTION1, OPTION2, OPTION3, OPERATION_TYPE

gee6809 reviewed (approved):
LGTM

Bammuri reviewed (approved):
저도  MAX 부분 형식 통일 외엔 괜찮을꺼 같습니다.

jieon-seol committed:
[Define] modify @OPTION3: MAX_OPTION3 -> MAX

jieon-seol committed:
[Define] add OPTION1,2,3 changes to EmployeeInfo
